drivers/hmis/lib/tasks/ce_define_workflows.rake

```diff
@@ -0,0 +1,186 @@
+desc 'Script to create CE workflow definition'
+# This task is for developing and iterating on CE workflow definitions.
+# It will be run in staging/training environments until the workflows are ready, at which point we will run it in production.
+# CAUTION: It deletes existing referrals and opportunities, so that we don't have to worry about definitions shifting underfoot.
+# This means it should NOT be run in production after the first time!
+# Usage: rails driver:hmis:ce_define_workflows
+task ce_define_workflows: [:environment] do
```

Comment:
Can we refactor this in a way that will make it easier to define multiple templates in the task? We will likely end up with more than one template.

For example, pulling out a function for `def delete_template_and_associated_data(template_identifier)`, and maybe even moving specific templates to their own functions so we can more easily separate out the logic, like `def create_workflow_template_whatever_its_called!`

Reply:
Yes, I think this refactor is nicer and hopefully easier to develop with. Thanks!